pub/main-app-screenshot.sh

```
@@ -196,62 +365,90 @@ if [[ -n "$AGENCY_TIME_ZONE" ]]; then
 fi
 
-TIME_FORMAT=$($ADB shell settings get system time_12_24)
-echo "TIME_FORMAT:'$TIME_FORMAT'."
 AVD_NAME=$(adb shell getprop ro.boot.qemu.avd_name || "")
-echo "AVD_NAME:'$AVD_NAME'."
-if [[ "${LANG}" == "en-US"]]; then
-  if [[ "${TIME_FORMAT}" != "12" && -n "$AVD_NAME" ]]; then
-    $ADB -e shell settings put system time_12_24 12
+
+# change device main language and locale based on the selected language
+CURRENT_LANG=$($ADB shell getprop persist.sys.locale || "")
```

Shellcheck (0.11.0)

[error] This empty string is interpreted as a command name. Double check syntax (or use 'true' as a no-op).

(SC2286)